taxonomies/ingredients.txt

```diff
@@ -47879,6 +47951,10 @@ pl:Nasiona dyni, nasiona dyni
 en:organic pumpkin seed
 fi:luomu kurpitsansiemen
 
+<en:pumpkin seed
+en:chopped pumpkin seed
+es:semillas de calabaza troceadas
```

Comment:
add troceadas to the ingredients processing taxonomy instead

Reply:
ok